featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -330,6 +330,37 @@ def test_arithmetic_of_direct(es):
         assert v == test[1]
 
 
+def test_boolean_multiply():
+    es = ft.EntitySet()
+    df = pd.DataFrame({"index": [0, 1, 2],
+                       "bool": [True, False, True],
+                       "numeric": [2, 3, np.nan]})
+
+    es.entity_from_dataframe(entity_id="test",
+                             dataframe=df,
+                             index="index")
+
+    to_test = [
+        ('numeric', 'numeric'),
+        ('numeric', 'bool'),
+        ('bool', 'numeric'),
+        ('bool', 'bool')
+    ]
+    features = []
+    for row in to_test:
+        features.append(ft.Feature(es["test"][row[0]]) * ft.Feature(es["test"][row[1]]))
+
+    features.append(ft.Feature(es["test"]['numeric']) * 3)
```

Comment:
Is this feature tested?

Reply:
Sorry, I put that in as I was testing to make sure I didn't break anything with the MultiplyNumericScalar primitive and forgot to take it out. I'll remove it now as it shouldn't be part of this.